### PR TITLE
update pip to conda for quantecon package

### PR DESCRIPTION
**Pull request by mmcky**
**Note:**

There is still a need to introduce `pip` given `pandas-datareader` is a pip install in `pandas.rst` so I have amended `getting_started` to mention both pip and conda. 

source/rst/./getting_started.rst:Other libraries can be installed with ``pip`` or ``conda``.
source/rst/./pandas.rst:    !pip install --upgrade pandas-datareader

### CODE REVIEW

**Comment by mmcky** [comment]
this PR removes `quantecon` from `environment.yml` to ensure lectures are setup to bring in the `quantecon` package from `conda-forge`

**Comment by mmcky** [comment]
Decided to use `pkg/main` to simplify `conda` install. The `quantecon` package gets included in the `pkgs/main` channel within ~7-10 days of updating through conda-forge.

File: source/rst/getting_started.rst [hunk 1]
```diff
@@ -429,26 +429,25 @@ One library we'll be using is `QuantEcon.py <http://quantecon.org/quantecon-py>`
...
 
-    ``pip install --upgrade quantecon``
+    ``conda upgrade quantecon``
 
 Another library we will be using is `interpolation.py <https://github.com/EconForge/interpolation.py>`__.
 
 This can be installed by typing in Jupyter
 
-    ``!pip install interpolation``
+    ``!conda install -c conda-forge interpolation``
```
Comment (mmcky) [inline source/rst/getting_started.rst:450]: @jstac I changed this to use `conda` from `pip` for consistency but `interpolations` will need the conda-forge channel to be specified. Is this OK?

**Comment by jstac** [comment]
Yes, that's totally fine, thanks Matt.

Does `quantecon` come with a fresh install of Anaconda now?  Or does it need to be installed manually through conda?

If the former, perhaps don't need to tell people to install it specifically.

**Comment by mmcky** [comment]
thanks @jstac. The package appears in `pkgs/main` channel for `conda` but isn't listed as a package for the `anaconda` installer: https://docs.anaconda.com/anaconda/packages/py3.7_linux-64/ so it doesn't come with anaconda.

**Pull request merged**